configs/machines/albedo.yaml

```diff
@@ -1,138 +1,180 @@
-# The albedo.awi.de YAML config file
+################################################################################
+# The albedo.awi.de yaml config file for esm-tools
+#
+# P. Gierz, Nov/Dec 2022
+# M. Andres-Martinez, Nov/Dec 2022
+################################################################################
 name: albedo
-# Could be usefully extraced from /etc/os-distro or something
+# Could be useful if this is extraced from /etc/os-distro:
 operating_system: "linux-rocky"
-# Should not be defined here:
-#jobtype: compute NOTE(PG): ...why?? The computer configuration should not need to know anything about the jobtype.
 sh_interpreter: "/usr/bin/bash"
-
-
 # Batch system and configuration:
 batch_system: "slurm"
-accounting: false
+accounting: true
```

Comment:
I did not know Albedo will have accounts for the computing time... good to know.

Reply:
Just for reporting purposes; basically we want to have some way to reliably figure out how much compute time is being used by which group.

Reply:
so there you would just write your awi user right? not your project?